examples/lzapp-migration/tasks/solana/base58.ts

```diff
@@ -0,0 +1,36 @@
+import assert from 'assert'
+import fs from 'fs'
+import path from 'path'
+
+import { Keypair } from '@solana/web3.js'
+import bs58 from 'bs58'
+import { task } from 'hardhat/config'
+
+import { types as devtoolsTypes } from '@layerzerolabs/devtools-evm-hardhat'
+
+interface Base58FeesTaskArgs {
+    /**
+     * The path to the keypair file to be used.
+     */
+    keypairFile: string
+}
+
+assert(process.env.HOME != undefined, 'process.env.HOME needs to be defined')
```

Comment:
can you put this in the action?  Doing so outside of the function will make the assertion at load time, which can lead to confusing error messages.

Reply:
it's not in the action as it's needed as part of `defaultKeypairFile` which is used in the param (line 26)